sdk/src/main/scala/com/horizen/certificatesubmitter/strategies/WithKeyRotationCircuitStrategy.scala

```diff
@@ -44,15 +43,14 @@ class WithKeyRotationStrategy(settings: SidechainSettings, params: NetworkParams
       certificateData.endEpochCumCommTreeHash,
       certificateData.btrFee,
       certificateData.ftMinAmount,
-      scala.collection.JavaConverters.seqAsJavaList(certificateData.getCustomFields),
+      certificateData.getCustomFields.toList.asJava,
       signaturesBytes.asJava,
       certificateData.schnorrKeysSignaturesListBytes,
       params.signersThreshold,
       certificateData.previousCertificateOption.asJava,
       sidechainCreationVersion.id,
       certificateData.genesisKeysRootHash,
       provingFileAbsolutePath,
-      true,
```

Comment:
why this parameter disappeared?
Instead of removing it -> USE IT

Reply:
Done